spec/queries/get_activity_log_events_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+RSpec.describe GetActivityLogEvents, with_audited: true do
+  around do |example|
+    @now = Time.zone.local(2020, 2, 11)
+    Timecop.travel(@now) { example.run }
+  end
+
+  let(:provider_user) { create(:provider_user, :with_two_providers) }
+  let(:application_choices_for_provider_user) { GetApplicationChoicesForProviders.call(providers: provider_user.providers) }
+  let(:service) { GetActivityLogEvents.new(application_choices: application_choices_for_provider_user) }
+
+  let(:course_provider_a) { create(:course, provider: provider_user.providers.first) }
+  let(:course_provider_b) { create(:course, provider: provider_user.providers.second) }
+  let(:course_unrelated) { create(:course) }
+  let(:ratified_course_provider_b) { create(:course, accredited_provider: provider_user.providers.second) }
+  let(:ratified_course_unrelated) { create(:course, accredited_provider: create(:provider)) }
+
+  def create_application_choice_for_course(course)
+    course_option = course.course_options.first
+    course_option ||= create(:course_option, course: course)
+    create(:application_choice, :awaiting_provider_decision, course_option: course_option)
+  end
+
+  def create_audit_for_application_choice(application_choice, changes: {})
+    create(:application_choice_audit, user: provider_user, application_choice: application_choice, changes: changes)
+  end
+
+  describe '#call' do
+    it 'raises an error unless application_choices responds_to #to_sql ' do
```

Comment:
consider rewording this so it describes when something happens rather than when it doesn't e.g raises an error **when** application_choices **don'**t respond_to #to_sql